github/projects.yaml

```diff
@@ -128,6 +128,8 @@
   description: Ansible role to provide configuration Intrusion Detection Systems
 - project: ansible-security/ids_install
   description: Ansible role to install configuration Intrusion Detection Systems
+- project: ansible/galaxy-api
```

Comment:
For this to work, my account needs to be admin.  Until we create an ansible-zuul token, like we did for receptor